### DISCUSSION TITLE
带有滚动条的分组柱线图，柱子宽度与组间距与组内柱子间距之间的关系

### DISCUSSION THREAD

**Discussion by JuniorYoung** [antvis/G2#5521] [Q&A]
版本：4.2.7
Demo：https://riddle.alibaba-inc.com/riddles/64ffd17
问题1：分组柱线图中，组间距以及组内间距如何固定一个值，不随组内柱子数量变化而变化。
问题2：组内柱子均分该组宽度时，会将组中线的数据也视为一个柱子然后均分，比如一个组内有3个柱子加一条线，渲染时就会当做4个柱子平均分布。

麻烦大佬们帮忙看下这个问题 感谢

**Comment by pearmini** [comment]
1. 这个没有办法，只能设置柱子的宽度是一定的。
2. 不是把线的数据视为一个柱子，是因为每个日期最大有4个分组，只不过大部分日期都没有4个分组，所以会均分，但没有柱子。
